app/src/main/java/org/p2p/wallet/newsend/ui/search/NewSearchContract.kt

```diff
@@ -13,7 +13,7 @@ interface NewSearchContract {
         fun showUsers(result: List<SearchResult>)
         fun clearUsers()
         fun showUsersMessage(@StringRes textRes: Int?)
-        fun showLoading(isLoading: Boolean)
+        fun showLoading()
```

Comment:
I think we need use strategy pattern here, to avoid declare state methods here, we should use state class for it
@isakovch